lomond/constants.py

```diff
@@ -11,10 +13,12 @@
 # https://tools.ietf.org/html/rfc6455
 WS_VERSION = 13
 
-# Only report major.minor in version
-_version_identifier = '.'.join(__version__.split('.')[:2])
-
 # User agent sent with websocket request
-# Default will be DataplicityLomond/<major>.<minor
-# e.g "DataplicityLomond/0.1"
-USER_AGENT = 'DataplicityLomond/{}'.format(_version_identifier)
+USER_AGENT = 'Lomond/{} {}/{} {}/{} ({})'.format(
```

Comment:
could we have an additional placeholder here? What I mean is that if we have the agent, wormhole, ttng and router all using lomond then how do we filter out which client in particular is sending requests? if we would have additional field, we could assign some nice name like

```session = WebsocketSession('ws://example.com', client='dataplicity-agent')```

which would yield

```Lomond/0.1.7 CPython/2.7.13 Darwin/16.6.0 (Darwin-16.6.0-x86_64-i386-64bit dataplicity-agent```